pages.fr/common/alex.md

```diff
@@ -0,0 +1,21 @@
+# alex
```

Comment:
It might be worth mentioning that this tool only works in English

Reply:
why not! 
I added some words, and also fix some grammar errors ^^"